alert-database/src/main/java/com/synopsys/integration/alert/database/settings/SettingsKeyEntity.java

```diff
@@ -24,17 +24,28 @@
 
 import javax.persistence.Column;
 import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.SequenceGenerator;
 import javax.persistence.Table;
 
+import com.synopsys.integration.alert.database.BaseEntity;
 import com.synopsys.integration.alert.database.DatabaseEntity;
 
 @Entity
 @Table(schema = "alert", name = "settings_key")
-public class SettingsKeyEntity extends DatabaseEntity {
-    @Column
+public class SettingsKeyEntity extends BaseEntity implements DatabaseEntity {
+    private static final long serialVersionUID = 7078325297864692468L;
+    @Id
+    @GeneratedValue(generator = "alert.settings_key_id_seq", strategy = GenerationType.SEQUENCE)
+    @SequenceGenerator(name = "alert.settings_key_id_seq", sequenceName = "alert.settings_key_id_seq")
+    @Column(name = "id")
+    private Long id;
```

Comment:
Do we perform bulk inserts on this table? We might be able to get away with `IDENTITY` here.